### ISSUE TITLE
onnxmltools import fails when pyspark is not installed

### ISSUE THREAD

**Issue by xadupre** [onnx/onnxmltools#246]

**Comment by vinitra** [comment]
Thanks @xadupre! I was able to replicate this by installing onnxmltools from source as well -- python 3.6, windows, no pyspark.

```
>>> import onnxmltools
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\viswamy\ProgramData\Anaconda3\lib\site-packages\onnxmltools\__init__.py", line 28, in <module>
    from .utils import load_model
  File "C:\Users\viswamy\ProgramData\Anaconda3\lib\site-packages\onnxmltools\utils\__init__.py", line 20, in <module>
    from .sparkml_test_utils import *
  File "C:\Users\viswamy\ProgramData\Anaconda3\lib\site-packages\onnxmltools\utils\sparkml_test_utils.py", line 2, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'

```

@jeffsaremi, it might make sense to move the sparkml_test_utils.py file (and the import statement in utils/__init__.py) elsewhere since pyspark is not a requirement for users intending to convert other frameworks' models to ONNX. The tests/utils folder seems like a more natural place.

**Comment by wenbingl** [comment]
Agree, any python package is not installed by requirements.txt should be optional in the common code.

**Comment by vinitra** [comment]
Resolved in #250, thanks @jeffsaremi!
